src/utils/AkamaiTokenService.js

```diff
@@ -0,0 +1,150 @@
+const TOKEN_TYPES = {
+  AKAMAI: 'AKAMAI',
+  NONE: 'NONE',
+};
+
+class AkamaiTokenService {
+  /**
+   * Get the acl path.
+   *
+   * @param {URL} streamUrl
+   *
+   * @returns {String}
+   */
+  static aclPath(streamUrl) {
+    let acl;
+
+    if (streamUrl.href.includes('/hls/playingLive/')) {
+      acl = '/hls/playingLive/*';
+    } else {
+      const path = streamUrl.pathname;
+
+      acl = `${path.substring(0, path.lastIndexOf('/') + 1)}*`;
+    }
```

Comment:
Interesting, we have a [different way](https://github.com/SRGSSR/pillarbox-apple/blob/main/Sources/CoreBusiness/ResourceLoading/AkamaiResourceLoaderDelegate.swift#L23-L29) of building the token. Ours seems a bit more generic, this is probably something we should discuss.

Reply:
@defagos I prefer your solution, it looks cleaner! Let's discuss it on Monday. 

If I remember correctly, I asked the android devs how they generated their URLs and I applied the same rule to Letterbox, so I imported the same logic.

FYI, here's what Android does today in [pillarbox](https://github.com/SRGSSR/pillarbox-android/blob/5008a1015023cbff075b3427e5c1859a84e4f528/pillarbox-core-business/src/main/java/ch/srgssr/pillarbox/core/business/akamai/AkamaiTokenProvider.kt#L41-L46)

Reply:
Great, I guess you are both wrong then 😉. Let us discuss this implementation on Monday then.

Reply:
After the discussion we add, I removed the check for the `/hls/playingLive/` which might be some sort of legacy code. Thank you @defagos 👍🏽